ui/src/assets/widgets/button.scss

```diff
@@ -24,6 +24,7 @@
   padding: 4px 8px;
   white-space: nowrap;
   min-width: max-content;
+  color: white;
```

Comment:
Is this assuming dark mode by default? I'm not sure that works for stand-alone Perfetto.

Maybe this isn't even needed now that `.pf-minimal` overrides it.

Reply:
The color is overridden if the class contains pf-minimal to inherit the text color from its parent object 
```
  &.pf-minimal {
    color: inherit;
    background: $pf-minimal-background;
```

Reply:
That's a restatement of my question, not an answer. But as I cannot find any buttons in the UI that are showing white text, I guess we'll find out when we try to sync with upstream whether there are problems elsewhere in the UI and for our application it works as expected.

Reply:
In the event where this CSS selector is used the background is already set.  To reduce visual contrasting issues I had it control the text color as well.  In the situation where the minimal class is used the background is unset meaning that the background of the button ends up being whatever is behind the button.  This is why we inherit the text color rather than set it ourselves in this case.

Reply:
> I cannot find any buttons in the UI that are showing white text

The Show button on the popup uses the class with white text.  It seems like their theming CSS that they have setup in Perfetto was only used in a few cases, which is why we haven't had issues with this side of the code a lot.  My guess is that it was added later than a lot of key features, but some of the more recent feature like the SQL Querying from the searchbox adopted them.